ddp-workspace/projects/ddp-rgp/src/app/components/study-messages/study-messages.component.scss

```diff
@@ -23,6 +23,10 @@
   }
 }
 
+.show-more-container {
+  margin-top: 10px;
```

Comment:
This is based on padding for `p` tags, which in RGP looks like it uses hardcoded pixels.